### DISCUSSION TITLE
Unable to deploy skyvern within coolify.

### DISCUSSION THREAD

**Discussion by Plaidmustache** [Skyvern-AI/skyvern#1276] [Q&A]
I am plagued with issues when deploying skyvern within coolify. am using traefik as the default proxy, which works with my other apps, but skyvern doesnt seem to handle it for some reason. ive asked two separate people to help me with it, both who cannot get it working. I am seeking help, if anyone is willing to take a look at my setup, i'd be willing to pay for your time to get skyvern running within my coolify, and also secure with user authentication.

**Comment by suchintan** [comment]
Are you open to sharing a little bit more about what issues you run into? We're happy to help here but need more information to get started!!

You can also (privately) send me information at suchintan@skyvern.com if you'd like

**Comment by tylerdurden4285** [comment]
> I am plagued with issues when deploying skyvern within coolify. am using traefik as the default proxy, which works with my other apps, but skyvern doesnt seem to handle it for some reason. ive asked two separate people to help me with it, both who cannot get it working. I am seeking help, if anyone is willing to take a look at my setup, i'd be willing to pay for your time to get skyvern running within my coolify, and also secure with user authentication. 

Did you find any solution to get it working?

**Comment by olsonbd** [comment]
I managed to get a fork deployed on Coolify--environment variables are a little tricky though. Check it out: https://github.com/olsonbd/skyvern-coolify
